### ISSUE TITLE
🐛 Error stack trace line numbers are incorrect

### ISSUE THREAD

**Issue by rivy** [denoland/deno#21106]
I've run into stack traces with incorrect line numbers for certain source file constructions involving code blocks.

Here's a minimal example...

```typescript
// t-error.ts
import { assert as _ } from 'https://deno.land/std@0.205.0/testing/asserts.ts';
{
	throw new Error('fail');
}
```

```shell
deno run --quiet t-error.ts
error: Uncaught Error: fail
Warning Couldn't format source line: Column 11 is out of bounds (source may have changed at runtime)
    at file:///mnt/c/.../t-error.ts:3:11
```

The incorrect line numbers occur under POSIX and WinOS.

In bisecting the problem; it looks like the regression started back in Deno v1.23.0 (and continues up to my current version, Deno v1.37.2).

Related to #20643 and/or #20993?

**Comment by lucacasonato** [comment]
This was already fixed in last release (1.38.0) - https://github.com/denoland/deno_ast/pull/182

**Comment by rivy** [comment]
@lucacasonato , thanks for the (prior) fix and response.
